lighthouse-core/test/chromium-web-tests/download-depot-tools.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-set -euo pipefail
+set -eo pipefail
```

Comment:
otherwise you never fall into the helpful error message when the env variable is unset :)